invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -143,6 +143,69 @@ class FundingSchema(Schema):
     funder = fields.Nested(FunderL10NItemSchema)
 
 
+def compute_publishing_information(obj, dummyctx):
+    """Computes 'publishing information' string from custom fields."""
+
+    attr = "custom_fields"
+    field = obj.get(attr, {})
+    publication_date = obj.get("metadata", {}).get("publication_date")
+    publisher = obj.get("metadata", {}).get("publisher")
+
+    # Retrieve publishing related custom fields
+    journal = field.get("journal:journal")
+    imprint = field.get("imprint:imprint")
+    thesis = field.get("thesis:university")
+
+    result = {}
+    if journal:
+        title = journal.get("title", "")
+        if title:
+            volume = journal.get("volume", "")
+            issn = journal.get("issn", "")
+            issue = journal.get("issue", "")
+            pages = journal.get("pages", "")
+            journal_string = (
```

Comment:
I think it makes sense to have in the ui serializer, since it will be used in both the record landing page and search results.

I don't think we want the ISSN to show up here. Makes more sense under the "Imprint" field, if it's going in the top box.

One solution to avoid having to think too much about the implementation is to pick a citation style, and have this string match what is automatically generated in the citation box. There are lots of ways for this string formatting to go wrong. 

An example bug in the current Zenodo implementation. An article could have a page range, or an article number. In the current implementation those are both dropped into the pages field, but they need to be rendered differently (p. for an article number, pp. for page range). You can see an example of this at https://zenodo.org/record/1155339. If you change the automatic citation to 'IEEE', you'll see how the automatic citation treats the pages field.

Reply:
to be discussed internally

Reply:
> I think it makes sense to have in the ui serializer, since it will be used in both the record landing page and search results.
> 
> I don't think we want the ISSN to show up here. Makes more sense under the "Imprint" field, if it's going in the top box.
> 
> One solution to avoid having to think too much about the implementation is to pick a citation style, and have this string match what is automatically generated in the citation box. There are lots of ways for this string formatting to go wrong.
> 
> An example bug in the current Zenodo implementation. An article could have a page range, or an article number. In the current implementation those are both dropped into the pages field, but they need to be rendered differently (p. for an article number, pp. for page range). You can see an example of this at https://zenodo.org/record/1155339. If you change the automatic citation to 'IEEE', you'll see how the automatic citation treats the pages field.

@tmorrell to follow up on this: we agreed we need to keep the Zenodo citation style for the time being, to keep the backwards compatibility with the format which was discussed and agreed previously. If you would like to suggest a change here, I think it should be discussed on the telecon.